src/components/Header/Navbar.tsx

```diff
@@ -34,17 +37,13 @@ export default function ButtonAppBar() {
 		<Box sx={{ flexGrow: 1 }}>
 			<AppBar position='static'>
 				<Toolbar>
-					<HomeIcon
-						onClick={HomeHandler}
-						sx={{ marginRight: 3, cursor: 'pointer' }}
-					/>
 					<IconButton
 						size='large'
 						edge='start'
 						color='inherit'
 						aria-label='menu'
 						sx={{ mr: 3 }}>
-						<MenuIcon />
+						<HomeIcon onClick={HomeHandler} sx={{ cursor: 'pointer' }} />
```

Comment:
1. HomeHandler should be written with camelCase and is a function, so needs imperative naming - handleHome.
2. Navigate functions are duplicated from ListDividers functions - how about creating a custom hook that returns those? :)